src/algo/depth-to-rgb-calibration/optimizer.cpp

```diff
@@ -1525,6 +1563,69 @@ void optimizer::set_cycle_data(const std::vector<double3>& vertices,
     _dsm_params_cand_from_bin = dsm_params_cand;
 }
 
+void optimizer::enhanced_preprocessing()
+{
+    if (_settings.ambient == RS2_AMBIENT_LIGHT_NO_AMBIENT)
+    {
+        if (_z.width == 640 && _z.height == 480)
```

Comment:
There is a function that gets called in set_z_data() that is meant to change parameters based on resolution